runtime/src/accounts_background_service.rs

```diff
@@ -160,7 +160,7 @@ impl SnapshotRequestHandler {
                 // accounts that were included in the bank delta hash when the bank was frozen,
                 // and if we clean them here, the newly created snapshot's hash may not match
                 // the frozen hash.
-                snapshot_root_bank.clean_accounts(true, false);
+                snapshot_root_bank.clean_accounts(true, false, None);
```

Comment:
thinking we maybe want to be able to toggle this `last_full_snapshot_slot` based on a validator flag that we can use to toggle incremental snapshots on/off

Reply:
Sure, that works. Can you point me to code/documentation on how to do that?

Also, this change doesn't actually enable incremental snapshots; there's no functionality change. So if you'd like we can wait to add the toggle at the same time I add actual incremental snapshot support to ABS.

Reply:
>Can you point me to code/documentation on how to do that

It would pretty much be another option added here: https://github.com/solana-labs/solana/blob/v1.7.9/core/src/validator.rs#L91, and then plumbed through the system

>Also, this change doesn't actually enable incremental snapshots; there's no functionality change. So if you'd like we can wait to add the toggle at the same time I add actual incremental snapshot support to ABS.

That's fine!

Reply:
Thanks! Yes, I'll be adding incremental snapshot config stuff to the SnapshotConfig struct, which'll then go in to the validator CLI args. That'll handle toggling incremental snapshots on and off. And I'll default to "off" to begin with.